termdashdemo/termdashdemo.go

```diff
@@ -18,7 +18,9 @@ package main
 
```

Comment:
Thank you for adding this, this allowed me to test both and get a comparison. It looks like tcell might be running in a different color mode than termbox (or the colors are misaligned). Please look at the following two screenshots. I am on MacOs so it might be specific to this platform. Are you also seeing this difference? How can we address it?

Termbox:
<img width="1440" alt="termbox" src="https://user-images.githubusercontent.com/5315344/75636069-a01d3e00-5be9-11ea-83d5-63c3adfb7770.png">

Tcell:
<img width="1440" alt="tcell" src="https://user-images.githubusercontent.com/5315344/75636072-a7dce280-5be9-11ea-8bda-5a370687a644.png">

Reply:
Do you have anything in mind on how we could tackle the colour differences between termbox and tcell?